src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionxpathfilter/InputSuppressionXpathFilterEscapeChar.java

```diff
@@ -1,17 +1,22 @@
-package com.puppycrawl.tools.checkstyle.filters.suppressionxpathfilter;
-
 /*
- * Config: Default.
- */
-public class InputSuppressionXpathFilterEscapeChar {
+SuppressionXpathFilter
+file = (file)InputSuppressionXpathFilterEscapeChar.xml
+optional = (default)false
 
-    char a = '&'; // violation
+com.puppycrawl.tools.checkstyle.checks.AvoidEscapedUnicodeCharactersCheck
+allowEscapesForControlCharacters = (default)false
+allowByTailComment = (default)false
+allowIfAllCharactersEscaped = (default)false
+allowNonPrintableEscapes = (default)false
 
-    char b = '\"'; // violation
+*/
+package com.puppycrawl.tools.checkstyle.filters.suppressionxpathfilter;
+
+public class InputSuppressionXpathFilterEscapeChar {
 
-    char c = '\''; // violation
+    char escapedChar1 = '\u03bc'; // filtered violation
 
-    char d = '<'; // violation
+    char escapedChar2 = '\u0055'; // filtered violation
```

Comment:
Please explain this update, we need test how it works on symbols like this.
Nobody do ` '\u0055';` in real code.

I do not expect such change.

Reply:
Please see the comment below.